modules/users/client/controllers/signup.client.controller.js

```diff
@@ -21,7 +21,7 @@
     vm.suggestionsLimit = 3; // How many tribes suggested (including possible referred tribe)
 
     // Variables for invitation feature
-    vm.invitationCode = '';
+    vm.invitationCode = $location.search().code;
```

Comment:
Also add `&code` to here:

https://github.com/Trustroots/trustroots/blob/master/modules/users/client/config/users.client.routes.js#L261